### PR TITLE
Implement tokens

### PR DESCRIPTION
**Pull request by jmacaluso711**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

 Replaces sass vars with the tokens from ids-identity (soho theme, soho dark theme, soho high contrast theme) 

Tokenized vars:
 - input
 - buttons
 - checkbox
 - radio
 - dropdown
 - notifications (fka toast/popovers)
 - alerts / pill / tag (fka badges/tags)

**Related github/jira issue (required)**:

related to https://github.com/infor-design/design-system/issues/158

**Steps necessary to review your pull request (required)**:

1. pull branch
2. `npm install` to get latest `ids-identity` (1.1.0)

### CODE REVIEW

**Comment by tmcconechy** [comment]
screen shot 2018-07-16 at 10 02 36 am
screen shot 2018-07-16 at 10 03 24 am

**Comment by jmacaluso711** [comment]
Just pushed a fix for this.  You'll need to `npm install ids-identity@latest` (v1.1.1) to see the change.

**Comment by tmcconechy** [comment]
Cool should we update the pr to 1.1.1 then in the package.json ?

**Comment by jmacaluso711** [comment]
oh yeah.. sorry, forgot to push

File: src/core/_config.scss [hunk 1]
```diff
@@ -414,12 +414,12 @@ $cardlist-text-color: $font-color;
 
 // Checkboxes
 $checkbox-color: $color-primary;
-$checkbox-hover-border-color: $graphite08;
-$checkbox-disabled-font-color: $graphite04;
-$checkbox-disabled-select-bg-color: $graphite03;
-$checkbox-disabled-arrow-color: #f4f4f4;
-$checkbox-focus-shadow: 0 0 0 1px $body-bg-color-primary, 0 0 0 2px $checkbox-color, $focus-box-shadow;
-$checkbox-label-color: $input-color;
+$checkbox-hover-border-color: $checkbox-hover-border-color;
```
Comment (clepore) [inline src/core/_config.scss]: You do not need to lines like this. If `$checkbox-hover-border-color` is coming through from IDS, this statement is  a duplicate.

**Pull request merged**